tests/compile-fail/overflow_check_conditional.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+#![deny(overflow_check_conditional)]
+
+fn main() {
+	let a: u32 = 1;
+	let b: u32 = 2;
+	if a + b < a {
+
+	}
```

Comment:
@mcarton yeah was talking about that :smile: Getting this error while compiling [https://pastebin.mozilla.org/8862494](https://pastebin.mozilla.org/8862494). Should I remove `use syntax::ast::*;` completely? What's the difference between the two?